docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -444,6 +444,14 @@ image::images/ci-cd-canary-deployment-pipeline.png[Canary Deployment Pipeline]
 
 
 The primary HTTP API to check the health of a service is compare the transaction error rate with a threshold value.
+The API call requires authentication, we recomend to use an API Token to authenticate.
+The URL is `KIBANA_URL//internal/apm/services`, it is desing to query some service metrics.
+In order to check the transaction error rate, we have to pass some parameters to the API call.
```

Comment:
```suggestion
In order to check the transaction error rate of the service on the newly deployed instance, we have to pass the following parameters to the API call:
* `start` and `end`: time interval using the ISO-8601 format
* `kuery`: used to filter on the service name and for example the new version being deployed to narrow down to the canary instances. Example `service.name:"MY_SERVICE_NAME" AND service.version:"1.2.3"`
* `environment`: the environment on which the canary instances are deployed. Example: `production`.
```

It would be great to have an example of an ISO-8601 date

@kuisathaverat does this style of documentation work for you? This would replace the lines below the lines below.